src/main/java/friday/storage/Storage.java

```diff
@@ -0,0 +1,58 @@
+package friday.storage;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import friday.fridayexceptions.FridayException;
+
+public class Storage {
+    private static String filepath;
+
+    /**
+     * Loading tasks from the file and saving tasks in the file
+     * @param filepath The filepath to be loaded.
+     */
+    public Storage(String filepath) {
+        this.filepath = filepath;
+    }
+
+    /**
+     * Locates the TaskList file, creating it if it does not exist.
+     * @return The tasks within the TaskList file.
+     * @throws FridayException When a TaskList file cannot be created.
+     */
+    public ArrayList<String> loadFile() throws FridayException {
+        try {
+            File f = new File(filepath);
+            if (f.createNewFile()) {
+                throw new FridayException("TaskList not found, creating new TaskList file");
+            } else {
+                ArrayList<String> temporaryFile = new ArrayList<>();
+                Scanner s = new Scanner(f);
+                while (s.hasNext()) {
+                    temporaryFile.add(s.nextLine());
+                }
+                return temporaryFile;
+            }
+        } catch (IOException e) {
+            throw new FridayException("an error occured");
+        }
+    }
+
+    /**
+     * Saves all tasks into the TaskList file.
+     * @param allTasks The tasks to be saved into the TaskList file.
+     * @throws IOException When the tasks cannot be written into the TaskList file.
+     */
```

Comment:
```suggestion
    /**
     * Saves all tasks into the TaskList file.
     *
     * @param allTasks The tasks to be saved into the TaskList file.
     * @throws IOException When the tasks cannot be written into the TaskList file.
     */
```
Need a line between description and parameters. I think all your Javadocs have this error.  Otherwise this is a pretty good Javadoc. 👍